### PR TITLE
Add groups

### PR DESCRIPTION
**Pull request by zachintosh**

### CODE REVIEW

File: src/background/handlers/windows.ts [hunk 1]
```diff
@@ -2,13 +2,20 @@ import { Command } from '../../types'
 
 const classifier = 'window'
 
+const group = {
+  icon: '⚙️',
+  label: 'Browser',
+  char: '>',
```
Comment (kyle-west) [inline src/background/handlers/windows.ts:8]: This is the same char as chrome?

Comment (zachintosh) [reply src/background/handlers/windows.ts:8]: Yes, that's intentional. I figured the windows and chrome commands could all fall under the same group. What do you think?

Comment (kyle-west) [reply src/background/handlers/windows.ts:8]: hmm then why do they have different Icons? Wouldn't that lead to confusion?

Comment (zachintosh) [reply src/background/handlers/windows.ts:8]: Oh, that's a mistake. Eh, let's fix it later.

**Pull request merged**